src/Nri/Ui/Button/V6.elm

```diff
@@ -0,0 +1,813 @@
+module Nri.Ui.Button.V6 exposing
```

Comment:
Thanks for updating button!

I don't want to force the scope of your PR to balloon in size, but I think I just realized that Button.V5.copyToClipboard may not work - can we make sure that the copyToClipboard button works ok in V6?